Readability.js

```diff
@@ -1703,6 +1703,17 @@ Readability.prototype = {
    */
   isProbablyReaderable: function(helperIsVisible) {
     var nodes = this._getAllNodesWithTag(this._doc, ["p", "pre"]);
+    if (this._doc.querySelectorAll && Array.from) {
+      nodes = Array.from(nodes);
+      var brs = this._doc.querySelectorAll("br:nth-of-type(5)");
```

Comment:
I would at least leave a comment explaining that 5 is a bit of an arbitrary value here.